unity/ComputerVision-UnityClient/Assets/Scripts/AppManager.cs

```diff
@@ -238,30 +245,29 @@ void DrawRectangles()
                     {
                         height = imgScalingFactor * (faceDetectionRects[i][3] - faceDetectionRects[i][1]);
                         width = imgScalingFactor * (faceDetectionRects[i][2] - faceDetectionRects[i][0]);
-                        GUI.DrawTexture(new Rect(faceDetectionRects[i][0] * imgScalingFactor, faceDetectionRects[i][1] * imgScalingFactor, width, height), rectTexture, ScaleMode.ScaleToFit, true, width / height);
+                        GUI.DrawTexture(new Rect(faceDetectionRects[i][0] * imgScalingFactor, faceDetectionRects[i][1] * imgScalingFactor, width, height), faceRectTexture, ScaleMode.ScaleToFit, true, width / height);
                     }
                     break;
 
                 case ServiceMode.FaceRecognition:
                     height = imgScalingFactor * (faceRecognitionRect[3] - faceRecognitionRect[1]);
                     width = imgScalingFactor * (faceRecognitionRect[2] - faceRecognitionRect[0]);
-                    GUI.DrawTexture(new Rect((faceRecognitionRect[0] * imgScalingFactor), faceRecognitionRect[1] * imgScalingFactor, width, height), rectTexture, ScaleMode.StretchToFill, true, width / height);
+                    GUI.DrawTexture(new Rect(faceRecognitionRect[0] * imgScalingFactor, faceRecognitionRect[1] * imgScalingFactor, width, height), faceRecognitionConfidenceLevel < 105 ? greenRectTexture : redRectTexture, ScaleMode.StretchToFill, true, width / height);
```

Comment:
is confidence not a percentage? seems strange that less confidence == green texture

Reply:
<img width="928" alt="Screen Shot 2020-08-26 at 10 58 12 AM" src="https://user-images.githubusercontent.com/24863504/91339290-1eb6fa80-e78b-11ea-812d-bf6b95e909ba.png">
According to @brucearmstrong  doc for FaceRecognition lower is the better and his threshold is 105 so I just copied that.

Reply:
Gotcha. Is this standard @brucearmstrong ?

Reply:
@franklin-huang-mobiledgex: Yes. That "confidence" level has always been confusing. The value was arrived at through trial and error. You've got to watch out, because for object detection, it is a normal percentage value.